Source/WebCore/css/html.css

```diff
@@ -316,31 +314,34 @@ ol {
     list-style-type: decimal;
     margin-block-start: 1__qem;
     margin-block-end: 1em;
-    margin-inline-start: 0;
-    margin-inline-end: 0;
     padding-inline-start: 40px;
 }
 
+dir {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+}
+
 li {
     display: list-item;
     text-align: match-parent;
 }
 
-/* FIXME: this should also match ::before::marker and ::after::marker but we don't support
-   this yet. When we do, we can remove the code specific to ::before and ::after in
-   RenderListItem::computeMarkerStyle(), see bugs.webkit.org/b/218897. */
-::marker {
-    unicode-bidi: isolate;
-    font-variant-numeric: tabular-nums;
-    white-space: pre;
-    text-transform: none;
+ul, menu {
+    counter-reset: list-item;
 }
```

Comment:
This will break the list-item counter without the presentational hints implemented. Can we undo this too?

Reply:
Done.